src/components/ui/ContribGrid.module.css

```
@@ -41,6 +41,36 @@
   box-shadow: 0 0 0 1px var(--bd-3);
 }
 
+/* Loading state — a gentle pulse on the empty grid so the skeleton reads as
+   "fetching" instead of "no activity at all." */
+.loading .cell {
+  pointer-events: none;
+}
+.cellLoading {
+  animation: contribPulse 1.6s ease-in-out infinite;
+}
+@keyframes contribPulse {
```

Stylelint (17.7.0)

[error] Expected keyframe name "contribPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)